pkger/parser_models.go

```diff
@@ -0,0 +1,321 @@
+package pkger
+
```

Comment:
nearly all of this is from the `models.go` file. Separating them out by their domain. These types are relevant to parsing. The service stateful types are in the `service_models.go` file listed below.